index.js

```diff
@@ -24,6 +24,19 @@ const injectPositions = Object.freeze({
     [2]: '↑',
 });
 
+const richInjectRoles = Object.freeze({
+    [0]: '⚙️ System',
```

Comment:
You can make these translatable by prepending the string with context.t function
```suggestion
    [0]: context.t`⚙️ System`,
```

Reply:
Tried to do that and got:
```
Uncaught ReferenceError: context is not defined
    at index.js:28:10
```
Guess, I have to define these objects somewhere else?

Reply:
Oh yeah you need to import it from getContext first at the top of the module

```js
const { t } = SillyTavern.getContext();
```